docs/reference/secrets-reference.md

```diff
@@ -0,0 +1,46 @@
+---
+id: secrets-reference
+title: Secrets YAML Reference
+---
+
+Recyclarr can substitute secret values within your YAML configuration files. This keeps sensitive
+values out of your configuration files and enables configuration sharing without you having to
+remember to manually redact each time.
+
+:::note
+
+Secrets functionality was added in `v3.0.0` and using them is optional.
+
+:::
+
+Specify secrets with a `secrets.yml` located in your [recyclarr configuration directory](/file-structure.md#appdata-directory).
```

Comment:
I'm so sorry to nitpick on you here. Do you mind keeping line length under 100? I understand sometimes this isn't possible. My personal preference in this case would be to break the link apart like this:

```md
Specify secrets with a `secrets.yml` located in your [recyclarr configuration directory][appdata].

[appdata]: /file-structure.md#appdata-directory
```

`markdownlint` is less than helpful here; it's supposed to allow URLs to exceed max line length when they're on their own line (like I've shown above) but not if they're part of a paragraph. I probably do not have it configured correctly.